content/src/content/jcr_root/apps/core/wcm/components/video/v1/video/video.html

```diff
@@ -0,0 +1,28 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2021 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License./apps/digitals2/general/components/pages/basepage/deferCSSLoading.html
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<div data-sly-use.videoModel="com.adobe.cq.wcm.core.components.models.Video"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     class="cmp-video">
+    <video class="cmp-video__video"
+           poster="${videoModel.posterImageReference ? posterImageReference : false}"
+           controls="${!videoModel.hideControl ? true : false}"
+           loop="${videoModel.loopEnabled ? true : false}"
+           autoplay="${videoModel.autoplayEnabled ? true : false}"
+           playsinline="${videoModel.autoplayEnabled ? true : false}"
+           muted="${videoModel.autoplayEnabled ? true : false}">
+        <source src="${videoModel.fileReference}"/>
```

Comment:
you must map this link manually, as source element are not processed through the CQ Link Transformer